Readme.md

```diff
@@ -84,44 +84,63 @@ the specifics for *pino-multi-stream*:
 
 [pinoapi]: https://github.com/pinojs/pino#api
 
-<a id="caveats"></a>
-## Caveats
+### pinoms.multistream(streams)
+
+Manually create a single `multistream` as used internally by the
+wrapper:
+
+```js
+var fs = require('fs')
+var pino = require('pino')
+var multistream = require('pino-multi-stream').multistream
+var streams = [
+  {stream: fs.createWriteStream('/tmp/info.stream.out')},
```

Comment:
they have the same API of bunyan, and the tests of this module are still ok.

Doing what you want is going to be _very expensive_, as we can currently implement this by doing a pre-sorting.